packages/core/src/docdb/commands/addRegion.ts

```diff
@@ -89,7 +124,11 @@ export async function addRegion(node: DBClusterNode): Promise<void> {
             getLogger().info('Created cluster: %O', newCluster)
             void vscode.window.showInformationMessage(localize('AWS.docdb.addRegion.success', 'Region added'))
 
-            node?.parent.refresh()
+            if (node instanceof DBClusterNode) {
+                node?.parent.refresh()
+            } else {
+                node?.refresh()
+            }
```

Comment:
Nit: Should this be encapsulated in the class?

Reply:
I suppose I could move all the logic into the node classes, but it wouldn't make sense to encapsulate just this part.